test/3_records_test.js

```diff
@@ -382,4 +382,16 @@ contract("LinniaRecords", (accounts) => {
       assert.equal(tx3.logs[0].event, "LogRecordAdded")
       })
     })
+  describe("destructible", () => {
+    it("should not allow non-admin to destroy", async () => {
+      await assertRevert(instance.destroy({ from: accounts[1] }))
+    })
+    it("should allow admin to destroy", async () => {
+      const admin = accounts[0]
+      assert.notEqual(web3.eth.getCode(instance.address), '0x0')
+      const tx = await instance.destroy({from: admin})
+      assert(!tx.logs.length, `did not expect logs but got ${tx.logs}`)
```

Comment:
If there's no log you don't assert that there's no log. This can potentially go on forever, asserting things that don't happen (e.g. admin balance doesn't change)
If there is log you assert the name of the event and parameters.